src/item/duolingo.md

```diff
@@ -2,6 +2,6 @@
 title: 'Duolingo'
 link: 'https://www.duolingo.com'
 summary: 'Learn languages for free. Web or app.'
-tags: ['education', 'age-all']
+tags: ['education', 'apps' 'age-all']
```

Comment:
You're missing a comma here, so the build is failing ;)

Reply:
Added the comma, so this should now be working.